src/ocrd/decorators/__init__.py

```diff
@@ -148,8 +152,13 @@ def exit():
             s = io.StringIO()
             pstats.Stats(pr, stream=s).sort_stats("cumulative").print_stats()
             print(s.getvalue())
-        atexit.register(exit)
-    run_processor(processorClass, mets_url=mets, workspace=workspace, **kwargs)
+        atexit.register(goexit)
+    if log_filename:
+        log_ctx = redirect_stderr_and_stdout_to_file(log_filename)
```

Comment:
BTW this is one of the things I got rid of in the fork for our k8s deployment.

Reply:
No, but something we should discuss because that does not play well with k8s.

Reply:
Oh, I see.